.github/ISSUE_TEMPLATE/change_request.md

```diff
@@ -0,0 +1,72 @@
+## General description
+The general description of the change/feature, that will explain the core value and why it's needed.
+#### Prerequisites
+Links to any external dependency for this feature
+
+## API
+
+#### Design description
+
+A clear and concise description of the design proposition.
+
+- [ ] Is it backward compatible ?
+- [ ] Adds new API
+- [ ] Changes existing API
+- [ ] Removes existing API
+
+#### Changes
+
+Provide link to a PR or code snippet of old and new API.
+
+## Behavior
+
+#### Consideration
+List of everything to consider before writing test cases.
+
+- [ ] Does the suggested changes impact the current behaviour ?
+<!-- If yes, specify which behaviour will be changed and how or provide the link to a PR.-->
```

Comment:
I think you will not actually see these comments in a template itself

Reply:
You will see it on the template, not in the preview / post